### PR TITLE
Update install-agent.sh

### PR DESCRIPTION
**Pull request by upccup**

### CODE REVIEW

**Comment by xiaods** [comment]
+1

**Comment by vitan** [comment]
:+1:

File: install-agent.sh [hunk 1]
```diff
@@ -67,8 +67,21 @@ check_netcat()
 check_omega_agent() {
   if ps ax | grep -v grep | grep "omega-agent" > /dev/null
   then
-    echo "Omega Agent service running.Stop omega-agent"
-    service omega-agent stop > /dev/null 2>&1
+    echo "Omega Agent service is running now... "
+    echo "Wraning!!! Continue installation will overwrite the original version"
```
Comment (vitan) [inline install-agent.sh]: Wraning => Warning

echo "Warning!!! Continue to install will update the current version"

**Comment by vitan** [comment]
@upccup  看下comment

**Comment by upccup** [comment]
@vitan Update

**Pull request merged**